### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
### In this project, I finish creating the remaining classes for the school library and created the associations between them.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```
Comment (tufoinnkuo10) [inline book.rb:9]: - [ ] great job. please A method for adding rentals should be present in the book class. [learn more](https://github.com/microverseinc/curriculumruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
- [ ] kindly ensure the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

File: person.rb [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 class Person < Nameable
+  attr_reader :rentals, :id
+
   def initialize(age, name = 'unkown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
```
Comment (tufoinnkuo10) [inline person.rb:12]: - [ ] kindly ensure a  method for adding rentals should be present`
- [ ] also please the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by afanimartin — APPROVED** [review]
Hi @Loltolo-Lesapiti, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉 🎉 🎉 
- Changes made as requested

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**